build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -209,6 +209,8 @@ service ActionCache {
 // by the server. For servers which do not support multiple instances, then the
 // `instance_name` is the empty path and the leading slash is omitted, so that
 // the `resource_name` becomes `uploads/{uuid}/blobs/{hash}/{size}`.
+// To simplify parsing, a path segment cannot equal any of the following
+// keywords: `blobs`, `uploads`, `actions`, `actionResults`, and `size`.
```

Comment:
I don't think it's necessary to reserve 'size' (because it's only ambiguous when preceeded by 'blobs') but we probably should reserve 'operations' and 'capabilities'.

Reply:
Operations, agreed, done. But why capabilities? You think there's a chance they will have a resource name some time? I'm not sure about that.

Reply:
I think it's unlikely they would end up with a resource name, but given that the string 'capabilities' already appears in the URI, there's at least a chance. I don't think reserving it realistically hurts anyone, and it's easier to drop a restriction in the future than to add one.

Reply:
Sure, done.